lazy_tensor_core/lazy_tensor_core/csrc/LazyShapeDtype.cpp

```diff
@@ -65,7 +65,7 @@ std::vector<std::vector<int64_t>> compute_shape_index_select(const at::Tensor &
     const at::Tensor & index) {
   // Based on definition of https://pytorch.org/docs/stable/generated/torch.index_select.html.
   // Promote Rank 0 index tensor to a 1 * 1 tensor.
-  dim = at::maybe_wrap_dim(dim, self);
+  dim = at::maybe_wrap_dim(dim, self.dim());
```

Comment:
@alanwaketan can you comment on this?  I wasn't able to compile the code that is on staging branch, and it worked if I changed to this.  Am I missing something?

Reply:
Let me checkout your code and debug.

Reply:
Tried adding the header:
```
#include "aten/src/ATen/WrapDimUtils.h"
```